cmd/tidb-dashboard/main.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+// Copyright 2023 PingCAP, Inc. Licensed under Apache-2.0.
```

Comment:
I think we should handle this change in an individual PR, instead of mixing with another feature PR.